css/style.css

```diff
@@ -0,0 +1,4 @@
+p{
```

Comment:
#### Critical :x:
In this case, all text is red as especified, but instructions say that you must apply this style to all body tag like this
```css
body{ 
    color: red;
}
```